README.md

```diff
@@ -1,70 +1,132 @@
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+### Run tests
+To run tests, run the following command:
+- Track HTML linter errors run:
 
-### Analyzing the Bundle Size
+npx hint .
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+- Track CSS linter errors run:
 
-### Making a Progressive Web App
+npx stylelint "**/*.{css,scss}"
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+- Track JavaScript linter errors run:
 
-### Advanced Configuration
+npx eslint .
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
 
-### Deployment
+### Deployment <a name="deployment"></a>
+- Deployed with github pages
+- For more information about publishing sources, see "[About GitHub pages](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#publishing-sources-for-github-pages-sites)".
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+## :busts_in_silhouette: Authors <a name="authors"></a>
+:bust_in_silhouette: *Emmanuel Kipngeno*
+- GitHub: [@githubhandle](https://github.com/kkmanuu)
+- Twitter: [@twitterhandle](https://twitter.com/kkmanuu)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/emmanuel-kipngeno-879370242/)
 
-### `npm run build` fails to minify
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+## :telescope: Future Features <a name="future-features"></a>
+
+ - *Use React Router.*
+ - *Use React components..*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :star:️ Show your support <a name="support"></a>
+If you like this project give start.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :handshake: Contributing <a name="contributing"></a>
+Contributions, issues [page issues](https://github.com/kkmanuu/math-magicians/issues), and feature requests are welcome!
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :pray: Acknowledgments <a name="acknowledgements"></a>
+I would like to thank  Microverse for this opportunity.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :memo: License <a name="license"></a>
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
The link to the project license is broken. please let's fix that, let' 's avoid broken links in our readme file for a professional repository. ☺💪

Reply:
@abdulhamiid,

I am happy to inform you that I have successfully implemented all the requested changes, and everything is now in good order. The modifications include refactoring the code to use separate components for better organization and reusability, as well as adding prop-type validation for improved data handling. Additionally, I have updated the project to follow best practices, resolving any potential issues.